tests/main.md

```diff
@@ -329,11 +329,85 @@ val len : int = 5
 - : unit = ()
 ```
 
+Reading with read:
+
+```ocaml
+# let t = Uring.create ~queue_depth:2 ();;
+val t : '_weak6 Uring.t = <abstr>
+
+# let fd = Unix.openfile Test_data.path [ O_RDONLY ] 0;;
+val fd : Unix.file_descr = <abstr>
+# let b1_len = 3 and b2_len = 7;;
+val b1_len : int = 3
+val b2_len : int = 7
+# let b1 = Cstruct.create b1_len and b2 = Cstruct.create b2_len;;
+val b1 : Cstruct.t = {Cstruct.buffer = <abstr>; off = 0; len = 3}
+val b2 : Cstruct.t = {Cstruct.buffer = <abstr>; off = 0; len = 7}
+# Uring.read t fd b1 `Read ~file_offset:Int63.minus_one;;
+- : _[> `Read ] Uring.job option = Some <abstr>
+# Uring.read t fd b2 `Read ~file_offset:Int63.minus_one;;
+- : _[> `Read ] Uring.job option = Some <abstr>
```

Comment:
This isn't right, I think. uring doesn't promise to execute operations in order by default, though there is a link flag (`IOSQE_IO_LINK`) that can be used for this. Specifying the file offsets would fix it.